src/integrationtest/resources/com/epam/idea/rest/controller/tag/tag-entries.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<dataset>
+    <user id="1" username="Jack" email="Jack@test.com" password="password" creation_time="2014-08-02 00:00:00"/>
+    <idea id="2" title="Test title" description="Test description" creation_time="2014-08-03 00:00:00" modification_time="2014-09-04 00:00:00" rating="5" user_id="1"/>
+    <tag id="3" name="Programming" />
+    <idea_tag idea_id="2" tag_id="3"/>
+</dataset>
```

Comment:
Add new line at the end of the file. (You can configure your IDE to always add empty line at the end of the file).